test/multivariate.jl

```diff
@@ -31,4 +31,8 @@ end
     @test rleft ≤ 1e-5 && rright ≤ 1e-5
     # Note: DynamicPolynomials automatically expands p, and evaluation using
     # interval arithmetic gives a worse left bound than the factored expression.
+
+    x = enclose(p, dom, SumOfSquaresEnclosure(backend=SDPA.Optimizer))
+    @test inf(x) ≈ 0.0 atol = 1e-3
+    @test sup(x) ≈ 670.612 atol = 1e-3
```

Comment:
How do you read this?

Reply:
`sup(x)` is approximately equal to `670.612` within `1e-3` absolute tolerance. Basically the `@test` macro allows you to pass extra kwargs to the function you are evaluating, so that the same 

```
@test isapprox(sup(x), 670.612; atol=1e-3)
```

Reply:
maybe rewriting as 
```julia
@test inf(x) ≈ 0.0 atol=1e-3
```

would be clearer? Now that I relook at it it's not that clear it's a kwarg